.github/workflows/ci-terraform.yml

```diff
@@ -0,0 +1,21 @@
+name: Terraform Plan
+
+on:
+  push:
+    branches-ignore:
+      - master
+
+env:
+  TF_VAR_DISCORD_SEVER_ID: ${{ secrets.TF_VAR_DISCORD_SERVER_ID }}
+  TF_VAR_DISCORD_TOKEN: ${{ secrets.TF_VAR_DISCORD_TOKEN }}
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I don't think the `GITHUB_TOKEN` is required, we can remove this and it should still work.

Reply:
It was mentioned in the action repo for some reason

Reply:
We will know, I ran the workflow in this PR